src/compiler/commandLineParser.ts

```diff
@@ -716,7 +716,8 @@ namespace ts {
                         }
 
                         // Skip over any minified JavaScript files (ending in ".min.js")
-                        if (/\.min\.js$/.test(fileName)) {
+                        // Skip over dotted files and folders as well
+                        if (/\.min\.js$/.test(fileName) || /[\\/]\.[\w.]/.test(fileName)) {

```

Comment:
consider combining into one regex, and hosting it as a variable declaration outside the function call to avoid creating a new literal every time the function is called.